wire/peer_wire.csv

```diff
@@ -245,6 +245,8 @@ msgdata,update_add_htlc,cltv_expiry,u32,
 msgdata,update_add_htlc,onion_routing_packet,byte,1366
 tlvtype,update_add_htlc_tlvs,blinding_point,0
 tlvdata,update_add_htlc_tlvs,blinding_point,blinding,point,
+tlvtype,update_add_htlc_tlvs,endorsed,1
+tlvdata,update_add_htlc_tlvs,endorsed,endorsed,byte,
 msgtype,update_fulfill_htlc,130
```

Comment:
This needs to be added as a patch, since it's not in the bolts, eg. wire/extracted_peer_12_jamming.patch.

Reply:
Thanks, done!